src/dnsperf.c

```diff
@@ -480,6 +486,11 @@ setup(int argc, char** argv, config_t* config)
     perf_opt_add('v', perf_opt_boolean, NULL,
         "verbose: report each query and additional information to stdout",
         NULL, &config->verbose);
+    perf_opt_add('U', perf_opt_string, "doh_uri",
```

Comment:
We need to change this, can't use short opts for specific transports because then all the short opts are going to run out and `-M` is already taken in `resperf`.

I'm going to add support for long opts, as soon as that is done you can rebase and switch to that.

Reply:
Use can use `perf_long_opt_add()` now.

Reply:
@jelu - addressed and rebased, thanks for the perf_long_opt_add() addition.